{"name": "test_delete_user_with_incorrect_user_id_031", "status": "passed", "description": "\n    Удалить не существующего пользователя (не существующий user_id)\n\n    Ожидаемый результат:\n        Статус-код 404;\n        Время ответа сервера - не превышает 500ms;\n        Response header \"Content-Type\": \"application/json\"\n        Response header \"Connection\": \"keep-alive\"\n        В тексте ошибки указан отправленный нами \"user_id\"\n    ", "start": 1721415265097, "stop": 1721415265139, "uuid": "3cc853a9-52e3-4630-b731-81ce2b19fa10", "historyId": "28064a7d22cfee043b49b6703dc4d6f0", "testCaseId": "28064a7d22cfee043b49b6703dc4d6f0", "fullName": "tests.users.test_users#test_delete_user_with_incorrect_user_id_031", "labels": [{"name": "tag", "value": "users"}, {"name": "parentSuite", "value": "tests.users"}, {"name": "suite", "value": "test_users"}, {"name": "host", "value": "DESKTOP-55MLA3F"}, {"name": "thread", "value": "23412-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.users.test_users"}]}